steal-stache.js

```diff
@@ -5,21 +5,30 @@ var loader = require("@loader");
 var addImportSpecifiers = require("steal-config-utils/import-specifiers").addImportSpecifiers;
 
 function template(imports, intermediate, filename){
+	var tagImportNames = JSON.stringify(imports.slice(6));
 	imports = JSON.stringify(imports);
 	intermediate = JSON.stringify(intermediate);
 
-	return "define("+imports+",function(module, assign, stache, mustacheCore){ \n" +
+	return "define("+imports+",function(module, assign, stache, mustacheCore, canViewImport, canStacheBindings){ \n" +
 		(filename ?
 			"\tvar renderer = stache(" + JSON.stringify(filename) + ", " + intermediate + ");\n" :
 			"\tvar renderer = stache(" + intermediate + ");\n"
 		) +
+		"\tvar tagImports = Array.prototype.slice.call(arguments, 6);\n" +
 		"\treturn function(scope, options, nodeList){\n" +
 		"\t\tvar moduleOptions = assign({}, options);\n" +
+		"\t\tvar tagImportMap = " + tagImportNames + ".reduce((map, name, index) => {\n" +
+		"\t\t\tmap[name] = tagImports[index];\n" +
+		"\t\t\treturn map;\n" +
+		"\t\t}, {});\n" +
+		"\n"+
 		"\t\tif(moduleOptions.helpers) {\n" +
-		"\t\t\tmoduleOptions.helpers = assign({ module: module }, moduleOptions.helpers);\n" +
+		"\t\t\tmoduleOptions.helpers = assign({ module: module, tagImportMap: tagImportMap }, moduleOptions.helpers);\n" +
```

Comment:
this shouldn't be added as a helper, it should be added as a LetContext

Reply:
https://github.com/canjs/can-view-scope/blob/28b36610a020098aa26ddf641cb8314b7409d9bf/can-view-scope.js#L718

Reply:
A bit of discovery is probably needed here.  I'm not sure everywhere `let contexts` are normally made.  Here's one place:

https://github.com/canjs/can-stache/blob/bf87a5acc05dbb3a38513903807e22b39b1948a5/can-stache.js#L546

I feel like we're going to have to check the `scope` here and see if it's, in fact, an instance of Scope.  

If it is, we might want to add these imports to that scope's let context.

If not, we should create a scope and add our own let context with these imports.

The function listed above does much of the same thing.  If we end up re-creating it too much, we might want to export it from stache and re-use it in can-view-import.

Reply:
Should we also make it so `module` is passed as part of the let context rather than as a helper? Seems to make sense for the same reasoning as including `tagImportMap` as a let variable rather than a helper.

Reply:
What does module do? Is tagImportMap added directly? If it is, why is it?